packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -1092,7 +1092,7 @@ class TimeInput extends React.Component {
             size="2"
             pattern="\d*"
             description={`${hourDescription()}, ${hotKeyDescription}`}
-            errorId={errorId}
+            ariaDescribedBy={errorId}
```

Comment:
You can instead rename `ariaDescribedBy` prop in accessibleInput to `errorId` to keep it consistent. Since the description of that prop suggests its only used for err message announcement.

Reply:
Already i have updated to errorID in date picker and date time picker, but already kadali has added ariaDescribedBy prop in AccessibleInput component to read error message for date input(https://github.com/cerner/terra-framework/pull/1975/files#diff-f42c6c0190bcd6e255f09217576b27f9f78c66c4a407ea440f434bcfa0efb614R30).